manim/scene/scene.py

```diff
@@ -813,13 +812,15 @@ def wrapper(self, *args, **kwargs):
             if file_writer_config["skip_animations"]:
                 logger.debug(f"Skipping animation {self.num_plays}")
                 func(self, *args, **kwargs)
+                # If the animation is skipped, we mark its has as None. When sceneFileWriter will start combining partial movie files, it won't take into account None hashes.
+                self.animations_hashes.append(None)
+                self.file_writer.add_partial_movie_file(None)
```

Comment:
Comment: I see what you're doing here, and I'm fine with it, but this sounds like something that should change at some point. Not sure if None is the most meaningful here...

Reply:
What do you mean ? for me, None is a good choice, as kind of explicit.

Reply:
I had an alternative when I wrote the above comment, but now I don't remember it. :shrug: